ui/src/app/pages/workspace/research-purpose.tsx

```diff
@@ -57,7 +57,8 @@ const styles = reactStyles({
 export const ResearchPurpose = withCurrentWorkspace()(
   ({workspace}: {workspace: WorkspaceData}) => {
     const isOwner = WorkspacePermissionsUtil.isOwner(workspace.accessLevel);
-    const selectedResearchPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose);
+    const selectedResearchPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose, true);
+    const selectedPrimaryPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose, false);
```

Comment:
open for name suggestions : This should reflect all selected Primary purpose  that are not research purpose

Reply:
The name seems fine to me.